lib/beaker-pe/install/pe_utils.rb

```diff
@@ -148,10 +148,10 @@ def frictionless_agent_installer_cmd(host, opts, pe_version)
             end
           end
 
-          # pe_repo also supports flags that will determine what happens during the frictionless install
+          # PE 2018.1.0 introduced a pe_repo flag that will determine what happens during the frictionless install
           # Current support in beaker-pe is for:
           # --puppet-service-debug, when running puppet service enable, the debug flag is passed into puppt service
-          if host[:puppet_service_debug_flag] == true
+          if (host[:puppet_service_debug_flag] and ! version_is_less(pe_version, '2018.1.0')) == true
```

Comment:
Why is the `== true` necessary here? That should probably remain a check against the host hash value, and the version_is_less can just have the implied true check from the condiational, right?

Reply:
Argggggh, good catch. Just a typo from where I pasted in that logic.